lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -474,7 +476,10 @@
                 success: this.clear_errors_then(function() {
                     return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                 }),
-                error: statusAjaxError(function() {
+                error: statusAjaxError(function(response) {
+                    if (response.responseText === 'Specified module does not support rescoring.') {
```

Comment:
Typically, such dependency on an error message string creates tight coupling between backend and frontend code and is not advisable from API usage standards.

Either, (1) have the backend code return an error _code_ (in addition to a message) that is checked and dispatched (and translated to the desired error message) or (2) have the backend code control the error message so the frontend can simply display it.

Why option 1?  Because status codes are generally maintained for compatibility by APIs, while error messages are not typically considered as part of an API contract.

However, I would recommend option 2: just update the error message you desire on the backend.  So we don't have to maintain multiple error strings.